v3/tags_data_test.go

```diff
@@ -0,0 +1,35 @@
+package exif
+
+import (
+	"testing"
+
+	log "github.com/dsoprea/go-logging"
+)
+
+func TestGeotiffTags(t *testing.T) {
+	testImageFilepath := getTestGeotiffFilepath()
+
+	// Returns a slice starting with the EXIF data and going to the end of the
+	// image.
+	rawExif, err := SearchFileAndExtractExif(testImageFilepath)
+	log.PanicIf(err)
+
+	exifTags, _, err := GetFlatExifData(rawExif, nil)
+	if err != nil {
+		t.Errorf("unexpected %+v", err)
+	}
+
+	exifTagsIDMap := make(map[uint16]int)
+
+	for _, e := range exifTags {
+		exifTagsIDMap[e.TagId] = 1
+	}
+
+	if _, ok := exifTagsIDMap[0x830e]; !ok {
+		t.Fatalf("missing ModelPixelScaleTag!")
```

Comment:
`if exifTagsIDMap[0x830e] == 0` would be nicer.

Should be a proper statement since this isn't an actual error message. Please don't use exclamations (it's implied). Example: "Missing ModelPixelScaleTag.".